src/test/java/com/butbetter/storage/controller/StorageControllerTests.java

```diff
@@ -0,0 +1,120 @@
+package com.butbetter.storage.controller;
+
+import com.butbetter.storage.model.Address;
+import com.butbetter.storage.model.ProductInformation;
+import com.butbetter.storage.repository.ProductRepository;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+public class StorageControllerTests {
+
+    @Autowired
+    private StorageController controller;
+
+    @Autowired
+    private ProductRepository repository;
+
+    @BeforeEach
+    void setUp() {
+        repository.deleteAll();
+    }
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
if not needed, it can be removed

Reply:
auto generated

Reply:
if you don't need it, you don't need to generate it